src/displayapp/screens/CatDog.h

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include <lvgl/lvgl.h>
+#include <cstdint>
+#include "displayapp/screens/Screen.h"
+#include "displayapp/apps/Apps.h"
+#include <displayapp/Controllers.h>
+#include "Symbols.h"
+#define HORIZON   18000
+#define MIN_POWER 200
+#define MAX_POWER 500
```

Comment:
We try to avoid the preprocessor when possible. This can be a `constexpr` as well.